protocol/src/protocol/offline_acc.rs

```diff
@@ -0,0 +1,38 @@
+use crate::protocol::login::{Account, AccountImpl, AccountType};
+use rand::{distributions::Alphanumeric, Rng};
+pub struct OfflineAccount {}
+
+impl AccountImpl for OfflineAccount {
+    fn login(&self, name: &str, _password: &str, _token: &str) -> Result<Account, super::Error> {
+        let test = std::iter::repeat(())
+            .map(|()| rand::thread_rng().sample(&Alphanumeric) as char)
+            .take(20)
+            .collect();
+
+        Ok(Account {
+            name: name.to_string(),
+            uuid: Some(test),
```

Comment:
Is it possible to use None here instead of a random value, or does that cause issues?